src/views/Books/BookDetail.js

```diff
@@ -1,9 +1,10 @@
 import { useEffect, useState } from 'react';
 import Book from '../../components/book/Book';
 import { getBookById } from '../../services/books';
+import { Link } from 'react-router-dom';
 
-function BookDetail() {
-  const id = 1; // TODO: Use id from route
+function BookDetail(props) {
+  const id = props.match.params.id; // TODO: Use id from route
```

Comment:
We can also use the useParams hook to get the id
https://v5.reactrouter.com/web/api/Hooks